### PR TITLE
fix(service): update vercel api paths

### PR DESCRIPTION
**Pull request by ndom91**
We work with the Vercel API at work as a partner of theirs. They move fast and often have many versions in flight at once haha.

The paths you had configured weren't working anymore. These seemed to do the trick for me :+1:

### CODE REVIEW

File: services/vercel.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const axios = require("axios");
 
 module.exports = {
   async getDeployments() {
-    const res = await this._getClient().get(`/v5/now/deployments?projectId=${this._getApiProjectId()}&teamId=${this._getApiTeamId()}&limit=10`);
+    const res = await this._getClient().get(`/v6/now/deployments?projectId=${this._getApiProjectId()}&limit=10`);
     return res.data;
   },
   async getDeployment(id) {
-    const res = await this._getClient().get(`/v11/now/deployments/${id}?projectId=${this._getApiProjectId()}&teamId=${this._getApiTeamId()}`);
```
Comment (f3l1x) [inline services/vercel.js:9]: Current version of this endpoint is v11, according to https://vercel.com/docs/api#endpoints/deployments/get-a-single-deployment.

Did you try it with your change and v11?

**Comment by f3l1x** [comment]
Thx. Seems good to me drop teamId. But I see v11 of `Get a Single Deployment` in official vercel docs.

File: services/vercel.js (hunk 1 above)
Comment (ndom91) [reply services/vercel.js:9]: Yeah v11 didn't return anything. I checked my dev tools while browsing to the appropriate place in their webapp and there were multiple requests actually with the deploymentId, `v5` and `v7` I believe. One of them didn't work here so I went with the other that did return all the necessary data in the correct shape.

Comment (ndom91) [reply services/vercel.js:9]: Just double checked, seems to be `v10` and `v13` they're using atm.

**Comment by ndom91** [comment]
Okay I've removed the `teamId` stuff from a few more places then, and also updated the readme a bit. 

I'm relatively new to strapi and this was the first plugin i'd tried to use. I wasn't aware at first that you can create all those directories under `/admin/src/..` in order to customize the frontend yourself. Took me a while to figure out haha so I added a little not here in the readme for future newbies.

**Comment by johannessteu** [comment]
Could this be merged @f3l1x  ?

**Comment by f3l1x** [comment]
Bumped v0.0.5. Thank you

**Pull request merged**